articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,30 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone.  The default configuration right now can have atmost 15 UDs in multiple AZ nodetype. This huge number of UDs reduced the upgrade velocity. Using the new configuration the max UDs is reduced which results in faster updates, keeping intact the safety of the upgrades.   
+
+The update should be done via ARM template by setting the zonalUpdateMode property to “fast” and then modify a node type attribute such as adding a node and then removing the node to each nodetype.  The Service Fabric managed cluster resource apiVersion should be 2022-10-01-preview or later.
```

Comment:
can you please rephrase to explicitly mention that this update has to be done for every nodetype for making the new sceme effective. Then talk about how to do it ?